Program.cs

```diff
@@ -58,7 +58,10 @@ public class Program
         // *************************************************************************************************************************
 
         public static int Main(string[] args)
-        {
+        {
+
+            string connectionStringWithKey = "";
```

Comment:
Ahh, ok, I see what you're doing, you're actually modifying this file in the environment setup, before building.  I would recommend a slightly different approach - use a config file, modify the values in that, and then have the program code read from the config.  Modifying the code directly is a weird thing to do, particularly in a sample.